Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 108 hosting

## Related ticket:
[SCRUM-108](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-108?atlOrigin=eyJpIjoiZTFiODU2MzJlYTNlNDQ0Y2JjNDNhMGJhNGQzZGQxYTEiLCJwIjoiaiJ9)

## In this PR:
- Host Mautic in a private server
- Setup Mautic to track frontend
- Delete the local Mautic instance
- Host Frontend with Netlify
- Change Drupal database schema, change the collate to utf8mb4_unicode_ci
- Host Drupal with Pantheon
- Resolve CORS issue
- Update Readme file

## Testing instruction
- Check the live links in the readme file

[SCRUM-108]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
